tests/src/withdrawEL.test.js

```diff
@@ -0,0 +1,65 @@
+import "core-js/stable";
+import "regenerator-runtime/runtime";
+import {
+  waitForAppScreen,
+  zemu,
+  genericTx,
+  nano_models,
+  SPECULOS_ADDRESS,
+  txFromEtherscan,
+} from "./test.fixture";
+import { ethers } from "ethers";
+import { parseEther, parseUnits } from "ethers/lib/utils";
+import { ledgerService } from "@ledgerhq/hw-app-eth";
+
+const contractAddr = "0xe8ff2a04837aac535199eecb5ece52b2735b3543";
```

Comment:
⚠️ this is not a known address

Reply:
was a testnet goerli contract, it matches b2c.json will edit for mainnet value only